[2020-06-18 13:24:30,357] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: sh_preprensa_v1.save_processed 2020-06-18T17:17:27.152744+00:00 [queued]>
[2020-06-18 13:24:30,399] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: sh_preprensa_v1.save_processed 2020-06-18T17:17:27.152744+00:00 [queued]>
[2020-06-18 13:24:30,399] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 13:24:30,399] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-18 13:24:30,399] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 13:24:30,428] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): save_processed> on 2020-06-18T17:17:27.152744+00:00
[2020-06-18 13:24:30,432] {standard_task_runner.py:53} INFO - Started process 14171 to run task
[2020-06-18 13:24:30,549] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: sh_preprensa_v1.save_processed 2020-06-18T17:17:27.152744+00:00 [running]> strx
[2020-06-18 13:24:30,582] {logging_mixin.py:112} INFO - Saving processed...
[2020-06-18 13:24:30,614] {python_operator.py:114} INFO - Done. Returned value was: integrate_data
[2020-06-18 13:24:30,683] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=sh_preprensa_v1, task_id=save_processed, execution_date=20200618T171727, start_date=20200618T172430, end_date=20200618T172430
[2020-06-18 13:24:40,314] {logging_mixin.py:112} INFO - [2020-06-18 13:24:40,313] {local_task_job.py:103} INFO - Task exited with return code 0
